documentation/howto/os3_remote.adoc

```diff
@@ -1,107 +1,96 @@
 +
 .Set up Port Forwarding
-image::os_v3_dbserver_selectportforwarding.png[Set up Port Forwarding]
+image::os3_setupapp_portforward.png[Set up Port Forwarding]
 +
-. In the` Application Port Forwarding` window, click the **Find free local ports for remote ports** check box.
-. Click **Local Port** and then click **Start All**.
+. In the `Application Port Forwarding` window, click the `Find free local ports for remote ports` check box.
 
-**Result:** The `Status` column now shows `Started`, indicating that port forwarding is now active. Additionally, the `Console` view shows the status of port forwarding for the particular service.
+. Click `Start All`.
 
-[steam_app_logs]
-== Stream Application Logs
+`Result:` The `Status` column now shows `Started`, indicating that port forwarding is now active. Additionally, the `Console` view shows the status of port forwarding for the particular service.
 
-Application logs are general logs for an application running on a remote OpenShift 3 instance. The streaming application logs feature in JBoss Developer Studio is used to monitor applications and use the historical information to troubleshoot if the application fails or returns errors.
+.Start Port Forwarding
+image::os3_setupapp_start_portforward.png[Start Port Forwarding]
 
-==== Prerequisites
- - Ensure that OpenShift Client Binaries are set up (see <<set_binaries,Set up OpenShift Client Binaries>> for instructions).
+[[pod_logs]]
+== Stream Pod Logs
+
+Pod logs are general logs for an application running on a remote OpenShift 3 instance. The streaming application logs feature in the IDE is used to monitor applications and use the previous pod log to troubleshoot if the application fails or returns errors.
+
+`Prerequisite:` Ensure that the OpenShift Client Binaries are set up (see <<setup_binaries, Set up OpenShift Client Binaries>> for instructions).
 
 To stream the application logs:
 
-. In the `OpenShift Explorer` view, expand the connection and then expand **Pods**.
-. Right-click the relevant Pod and then click **Pod Log**.
+. In the `OpenShift Explorer` view, expand the project, the services, and then the Pods.
 
-image::os_v3_dbserver_selectpodlog.png[View the Pod Logs]
+. Right-click the relevant Pod and then click `Pod Log`.
 
-**Result:** The `Console` view displays the pod log.
+.Stream Pod Log
+image::os3_setupapp_podlog.png[Stream Pod Log]
 
-[stream_build_logs]
+`Result:` The `Console` view displays the Pod log.
+
+[[build_logs]]
 == Stream Build Logs
 
-Build logs are logs documenting changes to applications running on a remote OpenShift 3 instance. The streaming build logs feature in JBoss Developer Studio is used to view the progress of the application build process and to debug the application.
+Build logs are logs documenting changes to applications running on a remote OpenShift 3 instance. The streaming build logs feature in the IDE is used to view the progress of the application build process and to debug the application.
```

Comment:
actually build logs are logs of a build pod, i.e. a pod running an application build